cmd/list/externalauthprovider/cmd.go

```diff
@@ -82,7 +82,8 @@ func runWithRuntime(r *rosa.Runtime, cmd *cobra.Command) error {
 	}
 
 	if len(externalAuthProviders) == 0 {
-		return fmt.Errorf("there are no external authentication providers for this cluster")
+		r.Reporter.Infof("There are no external authentication providers for this cluster")
```

Comment:
Nit: do we usually tell the user what cluster is “this”?

Reply:
@mikeshng Refactored , would you mind taking another look?

Reply:
This one is LGTM. Please feel free to resolve thread.